File: seongwon/Programmers/Level_0/Lv0_문자열_정렬하기_(1).cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    for(int i = 0; i < my_string.size(); i++){
        if(my_string[i] >= 48 && my_string[i] <= 57){
            answer.push_back(my_string[i]-48);
        }
    }
    sort(answer.begin(), answer.end());
    return answer;
}

File: hyunji/Programmers/Level_0/lv0_문자열_정렬하기_(1).py
def solution(my_string):
    my_str = []
    
    for i in range(len(my_string)):
        if my_string[i].isdigit():
            my_str += my_string[i]
            
    my_str = list(map(int, my_str))
    my_str = sorted(my_str)
    answer = my_str
    
    return answer